src/libraries/restore/dirs.proj

```diff
@@ -4,8 +4,7 @@
   <ItemGroup>
     <Project Condition="'$(DotNetBuildFromSource)' != 'true'" Include="analyzers/analyzers.depproj" />
 
-    <!-- Build for all configurations -->
-    <Project Condition="'$(BuildAllConfigurations)' == 'true'" Include="netcoreapp/netcoreapp.depproj" />
+    <Project Condition="'$(DotNetBuildFromSource)' != 'true'" Include="netcoreapp/netcoreapp.depproj" />
```

Comment:
I guess that by doing this change, we can probably now remove a bunch of versionless (well I guess they are now netcoreapplatest) configurations in projects where we only wanted to build a netcoreapp 2.x implementation, but required a versionless one as well for the vertical, right?

Reply:
no, this will only restore netcoreapp3.1 ref-pack for the vertical.